Update documentation about testing (#2917)
segmentio/segment-docs: src/connections/sources/catalog/libraries/server/python/index.md

analytics.on_error = on_error
```

> If you don't want to send Segments informations while testing, you need to add in your test:

Reviewer: ```suggestion
If you don't want to send information to Segment during testing, add the following code to your test:
```

Author: Sorry, i'm not used to  github PR interface. Thanks for the review :)
